portfolio/src/main/java/com/google/sps/servlets/LoginServlet.java

```diff
@@ -27,25 +27,38 @@
 @WebServlet("/login")
 public class LoginServlet extends HttpServlet {
     private static final String LOG_PAGE_URL = "/login";
-
+    private static final String COMMENTS_URL = "/comments.html";
+    private static final String TEXT_TYPE = "text/html";
+    private boolean isLoggedIn;
 
     @Override
     public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-        response.setContentType("text/html");
+        response.setContentType(TEXT_TYPE);
         PrintWriter out = response.getWriter();
         UserService userService = UserServiceFactory.getUserService();
-
+            
         if(userService.isUserLoggedIn()) {
             String userEmail = userService.getCurrentUser().getEmail();
             String logoutURL = userService.createLogoutURL(LOG_PAGE_URL);
 
+//            response.sendRedirect(COMMENTS_URL);
             out.println("<p>Hello " + userEmail + "! You are logged in!</p>");
-            out.println("<p>Logout <a href=\"" + logoutURL + "\">here</a>.</p>");
+            out.println("<p>To view the comments, click <a href=\"" + COMMENTS_URL + "\">here</a>.</p>");
```

Comment:
I'm fine leaving this as-is, but just so you know, a more common paradigm is to not include HTML in your backend and only send the data itself to the UI, letting the Javascript/HTML handle actually creating the proper HTML elements.